app/Main.hs

```diff
@@ -22,7 +22,7 @@ import Backend.Interpreter
 import Error
```

Comment:
The `set-field` command still shows unqualified paths, see line 110 of `tests/golden/set-field-command/set-field-command.bats`

Maybe we should change `SFRSuccess Entry` to `SFRSuccess (QualifiedPath Entry)`?

Reply:
In general, I think most of the `*Result` constructors (`SetFieldResult`, `CreateResult`, `CreateError`, etc) that take an `Entry` should take a `QualifiedPath Entry` instead.

Here's another one that's missing:

```
$ coffer -c tests/golden/tag-command/config.toml create 'vault-local#/a'
$ coffer -c tests/golden/tag-command/config.toml create 'second#/a'     
$ coffer -c tests/golden/tag-command/config.toml copy 'vault-local#/a' 'second#/a'  
[ERROR] The following entries cannot be copied:

'vault-local#/a' to '/a':
  An entry already exists at '/a'.
  Use '--force' or '-f' to overwrite existing entries.
```

The 3rd and 4th lines should show `second#/a` instead of `/a`.

Reply:
Maybe some constructors could be simplified to take only a `QualifiedPath EntryPath` instead of an entire `QualifiedPath Entry`?

Reply:
Done